src/core_plugins/kibana/public/dashboard/panel/panel.html

```diff
@@ -34,6 +34,9 @@
     search-source="savedObj.searchSource"
     show-spy-panel="!isFullScreenMode"
     ui-state="uiState"
+    shared-item
```

Comment:
At one point, these were calling `reporting-item` to be very explicit towards their use; however, we switched the name to be more general because they're technically not just for reporting.